base/build.gradle

```diff
@@ -1,17 +1,22 @@
-apply plugin: 'com.android.application'
+apply plugin: 'com.android.feature'
 apply from: '../config/checkstyle.gradle'
+/**
+ * In order to use Butterknife in a Library module, we must do the following:
+ * https://github.com/JakeWharton/butterknife#library-projects
+ */
+apply plugin: 'com.jakewharton.butterknife'
+
 
 android {
-    compileSdkVersion 27
```

Comment:
you have changed compile version from 27 to 28. So, you got to update the build version here also: 
https://github.com/Swati4star/Images-to-PDF/blob/master/.travis.yml#L7

Reply:
I'm sorry, I created a new PR without knowing it!

https://github.com/Swati4star/Images-to-PDF/pull/501

Changed build tools and compile sdk

Reply:
You should make those changes a part of this PR